### PR TITLE
Setup origin-metrics 3.10 job and config

### PR DESCRIPTION
**Pull request by rubenvp8510**
@stevekuznetsov Could you please review?

Thanks

//cc @smarterclayton

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
/ok-to-test

@rubenvp8510 did you run the generator to create the entries under `ci-operator/jobs`?

**Comment by rubenvp8510** [comment]
@stevekuznetsov No, I didn't. Where I can find documentation on how to do it? Do I need some kind of permissions? Thanks

**Comment by petr-muller** [comment]
@rubenvp8510 The tool has a [README](https://github.com/openshift/ci-operator-prowgen) and the `ci/prow/generated-config` check gives a [hint](https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_release/1686/pull-ci-openshift-release-master-generated-config/142) about how to run it containerized to fix what it complains about.

If you followed some doc to create this PR, let us know which, we can mention the generator there.

**Comment by petr-muller** [comment]
I re-triggered the test because the hint I [mentioned above](https://github.com/openshift/release/pull/1686#issuecomment-425035414) was wrong and I have fixed it :)

**Comment by rubenvp8510** [comment]
@petr-muller @stevekuznetsov I've already generated the jobs based on the config and the README on  https://github.com/openshift/ci-operator-prowgen.

Also, did a rebase, but still failing, Any ideas?

File: ci-operator/jobs/openshift/origin-metrics/openshift-origin-metrics-release-3.10-postsubmits.yaml [hunk 1]
```diff
@@ -1,8 +1,34 @@
 postsubmits:
   openshift/origin-metrics:
-  - agent: jenkins
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/origin-metrics/openshift-origin-metrics-release-3.10-postsubmits.yaml]: Just to be sure, are you removing the jenkins-backed job intentionally?

Comment (rubenvp8510) [reply ci-operator/jobs/openshift/origin-metrics/openshift-origin-metrics-release-3.10-postsubmits.yaml]: yes, as far as I saw Jenkins job was not working for 3.10. so my intention with this PR is replacing the Jenkins job to build the images for metrics 3.10 with this.

Does that make sense?

Comment (petr-muller) [reply ci-operator/jobs/openshift/origin-metrics/openshift-origin-metrics-release-3.10-postsubmits.yaml]: Yeah, it totally makes sense, just wanted to make sure it is intentional (for example we might have a weird bug in the generator which is not supposed to touch jenkins jobs).

**Comment by petr-muller** [comment]
/lgtm
/approve

You may want to add an OWNERS file so your team can approve/lgtm these configs yourselves :)

**Comment by rubenvp8510** [comment]
@petr-muller @stevekuznetsov When this get merged, could you please trigger the job manually? Thanks.

**Comment by petr-muller** [comment]
@rubenvp8510 :ballot_box_with_check: https://openshift-gce-devel.appspot.com/build/origin-ci-test/logs/branch-ci-openshift-origin-metrics-release-3.10-images/2

**Pull request merged**